metasyn/var.py

```diff
@@ -45,6 +45,11 @@ class MetaVar():
         back.
     description:
         User-provided description of the variable.
+
+    Attributes
+    ----------
+    dtype : str
+        The data type of the original values. Default is "unknown".
```

Comment:
This can probably go now as well. The default should be part of the parameters though.